apps/test/unit/code-studio/pd/form_components_func/FormControllerTest.js

```diff
@@ -20,594 +21,567 @@ let DummyPage3 = () => {
 DummyPage3.associatedFields = [];
 
 describe('FormController', () => {
-  describe('Standard usage', () => {
```

Comment:
Removed this describe block, which encompassed the whole FormController describe